tests/Oryx.Tests.Common/ImageTestHelper.cs

```diff
@@ -346,5 +352,10 @@ private string GetTestTag()
 
             return _tagSuffix;
         }
+
+        private string GetFullBuildImage()
```

Comment:
nit: since we have to account for `full`, `full-buster` and `full-bullseye`, I think it would be best if we followed the same model that the above `GetGitHubActionsBuildImage` method uses to handle these different Debian flavors (_i.e._, provide a `debianFlavor` argument that we change the returned image name based on).

Also, we will need to account for the `_tagSuffix` at the end of the image name for tests in our pipeline that append the date-time to the end.